rollup.config.mjs

```diff
@@ -1,8 +1,6 @@
 import typescript from 'rollup-plugin-ts';
 import json from '@rollup/plugin-json';
-import peerDepsExternal from 'rollup-plugin-peer-deps-external';
 import { folderInput } from 'rollup-plugin-folder-input';
-import babel from '@rollup/plugin-babel';
```

Comment:
if you remove it, plz remove from package.json

also you keep babel config in root - remove